Brewfile

```diff
@@ -0,0 +1,4 @@
+tap 'pact-foundation/pact-ruby-standalone'
```

Comment:
To avoid having these changes that were not made by you in the PR I would encourage you to rebase `origin:master` onto your `fork:branchPR`, then force push (onto your `fork:branchPR`!!!) the new history. 

That way any of the changes made to origin will not show up in your PR.

Reply:
TODO: (for me)
yes, I agree. I'll have to see when I have time for this. Since I think this is what I did the last time.